docs/documentation/index.html

```diff
@@ -240,7 +229,7 @@ <h4>Source Group Setup for C/C++</h4>
 
 					<strong>Create Empty</strong>
 
-					<p>If either of the above options apply to your project, please continue at <a href="#CreateAnEmptyCXXProject">Create an empty C/C++ Source Group</a> and come back here as soon as the project is created.</p>
+					<p>If neither of the above options apply to your project, please continue at <a href="#CreateAnEmptyCXXProject">to create an empty C/C++ Source Group</a> and return here once the project is created.</p>
```

Comment:
I think it would be better to change it back from "to create" to "Create" because otherwise it won't read correctly with the link.

Reply:
Okay no problem. I'll make the change later today and re-commit.